hack/make-rules/verify/generated-jobs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Installing requirements3.txt"
+hack/run-in-python-container.sh \
+    pip3 install -r requirements3.txt
+
+echo "Verifying generated jobs"
+hack/run-in-python-container.sh \
+    python3 hack/generate-jobs.py config/jobs/kubernetes/sig-node/*.conf --only-verify
```

Comment:
We can start with only working on "our" (= SIG Node) jobs here for now. But in a future PR this should get extended to other generated jobs.

Reply:
Yes. Fortunately it's just a matter of adding new patterns to the command line. `generate-jobs.py` already supports this.

Reply:
Such a change would also have to be made over in hack/make-rules/update/generated-jobs.sh. There's a risk that someone adding a new config there forgets to add it here.

Can we perhaps use a pattern that is unique enough that we don't need to update it, for example `config/jobs/**/*.generate-conf`?

Reply:
https://github.com/kubernetes/test-infra/pull/34010#discussion_r1907319393

> Done. It required a bit of refactoring as Python glob.glob doesn't understand this kind of pattern.